src/telemetry/logs.ts

```diff
@@ -223,7 +227,7 @@ class OTLPLogQueueTransport extends TransportStream {
       timestamp: new Date().getTime(), // So far I don't see a major difference between this and observedTimestamp
       observedTimestamp: new Date().getTime(),
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-      attributes: { ...span?.attributes, traceId: span?.spanContext()?.traceId, spanId: span?.spanContext()?.spanId, stack } as LogAttributes,
+      attributes: { ...span?.attributes, traceId: span?.spanContext()?.traceId, spanId: span?.spanContext()?.spanId, stack, applicationID: this.applicationID, executorID: this.executorID } as LogAttributes,
```

Comment:
executorID is already present in the span attributes, when provided. I think your solution is nicer - so we do not have to add executorID to span attributes manually each time we create a new Span.

But we should remove the duplicate, e.g., https://github.com/dbos-inc/dbos-ts/blob/d47c99b9bd49ef9660d3cae5445e1be3687a8663/src/dbos-executor.ts#L387

Reply:
Hey Max! Yeah I looked at that one, briefly. It looks like that one comes in as part of the call header and is used in a few places. I wasn't sure if removing it would have side effects. But I can put this on my TODO list